packages/trader/src/Modules/Trading/Containers/smart-chart-switcher.jsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import { SmartChart } from 'Modules/SmartChart';
 import { SmartChartBeta } from 'Modules/SmartChartBeta';
+import LaunchModal from 'Modules/SmartChartBeta/Components/LaunchModal/launch-modal';
 
 const SmartChartSwitcher = ({ is_beta, ...props }) => {
     const Chart = is_beta ? SmartChartBeta : SmartChart;
-    return <Chart data-testid='SmartChart' {...props} />;
+    return (
+        <>
```

Comment:
@ahmadtaimoor-deriv we have convention to use React.Fragment instead of its shorthand :)
```suggestion
        <React.Fragment>
```